test/test_helper.rb

```diff
@@ -5,7 +5,7 @@
 
 $VERBOSE = true # enable ruby warnings
 
-require 'mocha/setup'
```

Comment:
```
Mocha deprecation warning at /home/travis/build/travisjeffery/timecop/test/test_helper.rb:8:in `require': Require 'mocha/test_unit', 'mocha/minitest' or 'mocha/api' instead of 'mocha/setup'.
```